web/components/home/Home.js

```diff
@@ -132,6 +132,9 @@ class Home extends React.Component {
   render() {
     return (
       <div className="homepage overflow-hidden flex jc-center ai-center dir-col">
+      <a className="mlh-trust-badge"  href="https://mlh.io/seasons/na-2019/events?utm_source=na-hackathon&utm_medium=TrustBadge&utm_campaign=2019-season&utm_content=black" target="_blank">
```

Comment:
Ideally we want to store our external links in `components/home/External.js`. You can find an example of it being used on [line 151](https://github.com/nwplus/nwhacks2019/blob/45867811c1a7e3c91118331723a3305901f07356/web/components/home/Home.js#L151).